ql/pom.xml

```diff
@@ -437,6 +437,29 @@
       <artifactId>opencsv</artifactId>
       <version>${opencsv.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Is there a need for this dependency ? Ideally this should not happen as this leaks storage handler specific logic in ql which breaks abstraction.